content/docs/support-and-ops/processes/customer-check-upgrade-changes.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "How to check for changes in k8s upgrades"
+owner:
+- https://github.com/orgs/giantswarm/teams/team-teddyfriends
+confidentiality: public
+---
+
+### Purpose
+
+When upgrading Giant Swarm releases you will notice that the kubernetes version is also being upgraded. As part of the upgrade process it should be a habit to check for the newest changes upstream. This ensures that the diffs, deprecations, and new features from an upstream kubernetes perspective are known. 
+
+There is not a right or wrong way to go about this process because there is not one tool that simply gives all of the answers. However, the following tools combine to give a nice overview of the important changes. Let us know if you have thoughts as well!  
```

Comment:
```suggestion
There is not a right or wrong way to go about this process because there is no single tool that simply gives all of the answers. However, the following tools combine to give a nice overview of the important changes. Let us know if you have thoughts or recommended tools as well!  
```